datasets/us/sam_exclusions/crawler.py

```diff
@@ -51,6 +53,7 @@ def crawl_data_url(context: Context) -> str:
 
 
 def crawl(context: Context) -> None:
+    reset_entity_links(context)
```

Comment:
When does this transaction get committed, i.e. will there be a time while the crawler is running when the links will be missing in the UI?

I'm wondering whether there are cases where we don't "touch" all reviews in a run (except for the failed run case). Would it be an option to clear them per-review when we touch that review? Or maybe we could add a `last_seen_version` like this the reviews itself and then somehow expire them? I haven't thought too deeply about this, just trying to think aloud.